### DISCUSSION TITLE
Adding a Network Meter

### DISCUSSION THREAD

**Discussion by Newer1107** [prasanthrangan/hyprdots#57] [Q&A]
![image](https://github.com/prasanthrangan/hyprdots/assets/116626403/afd5cd61-ab50-4eed-9904-87b428ada047)
I want to add a network speed meter right after the memory usage similar to the one you see after you hover over connections, except without the IP.
Does anyone has any idea about this?
Thanks in advance!

**Comment by prasanthrangan** [comment]
Hi, you can try updating `~/.config/waybar/modules/network.jsonc` as -

```
    "network": {
        // "interface": "wlp2*", // (Optional) To force the use of this interface
        "format-wifi": "󰤨 {essid}",
        "format-ethernet": "󱘖 Wired",
        "tooltip-format": "󱘖 {ipaddr}  {bandwidthUpBytes}  {bandwidthDownBytes}",
        "format-linked": "󱘖 {ifname} (No IP)",
        "format-disconnected": " Disconnected",
        "format-alt": "󰤨 {signalStrength}%",
        "interval": 5
    },

    "network#net2": {
        "format-wifi": " {bandwidthUpBytes}  {bandwidthDownBytes}",
        "format-ethernet": " {bandwidthUpBytes}  {bandwidthDownBytes}",
        "tooltip-format": "󱘖 {ipaddr}",
        "format-linked": "󱘖 {ifname} (No IP)",
        "format-disconnected": " Disconnected",
        "format-alt": "󰤨 {signalStrength}%",
        "interval": 5
    },
```

and use `network#net2` in `~/.config/waybar/config.ctl` to display the network speed.

**Reply by Newer1107** [reply]
Hello,
Thank you so much for helping
I really appreciate your assistance. I'll definitely try out the new additions once I'm home. 😊

**Comment by Newer1107** [comment]
![image](https://github.com/prasanthrangan/hyprdots/assets/116626403/4b4fc57a-db35-46af-a34d-7bebd344944f)
Hello!
Thanks for your help, it worked.
